pyproject.toml

```diff
@@ -60,9 +60,9 @@ simplegist = "^1.0.1"
 xmltodict = "^0.13.0"
 pymysql = "^1.1.1"
 configparser = "^7.0.0"
-ecs-logging = "^2.1.0"
+ecs-logging = "^2.2.0"
 celery = "^5.4.0"
-sqlalchemy = "^2.0.30"
+sqlalchemy = "^2.0.31"
```

Comment:
If we get rid of Matomo tracking completely (or move to Redis/RabbitMQ) we will be able to get rid of SQLAlchemy.  It was required for their connector to the database.